governance/SDK.md

```diff
@@ -0,0 +1,163 @@
+
+- Every SDK has its own
+  [Github Team](https://github.com/orgs/cloudevents/teams), for example
+  `sdk-go-maintainers` is the group of mantainers for
+  [cloudevents/sdk-go project](https://github.com/cloudevents/sdk-go)
+- The union of all the `sdk-*-maintainers` assembles the _sdk maintainers_ group
+
+## New maintainers
+
+TBD
+
+## Ensuring projects health
+
+In order to ensure the health of the SDK projects, we define the following
+actions. These are crucial to show the commitment of the community to develop
+and support these libraries, which are an important part of the CloudEvents
+ecosystem.
+
+We define an SDK project _healthy_ if:
+
+1. It works with the latest version of the programming language
+2. It supports the latest versions of the integrated libraries/frameworks
+3. It receives security patches regularly
+4. It supports the last N-1 major versions of CloudEvents spec, as defined in
+   [Contribution Acceptance](../SDK.md#contribution-acceptance)
+5. Issues and PRs are triaged (labeled, commented, reviewed, etc) regularly
+
+We define a project `cloudevents/sdk-x` _not actively maintained_ if:
+
+6. No commits on master from `sdk-x-maintainers` since 4 months
+7. Issues and/or PRs are not being triaged from `sdk-x-mantainers` since 2
+   months
+8. Security patches are not being **released** from `sdk-x-mantainers` since 1
+   months from CVE disclosures
+
+To prevent the project from becoming _not actively maintained_, the community MAY takes the following actions:
+
+- Temporary security patches delivered by an _sdk maintainer_ not part of
+  `sdk-x-maintainers` group
+- Handover of the project to a new maintainer
+- Archive the project
+
+### Security patches
+
+If a project `cloudevents/sdk-x` meets _healthy_ criteria 1, 2, 4, but the
+`sdk-x-maintainers` group is not actively performing security patches as defined
+in criteria 8, the community could decide to entitle one or more members from
+the larger _sdk maintainers_ group to perform security patches and release new
+minor versions of the project.
+
+Because the people entitled to perform security patches need write permissions on the repository, the community MUST temporarily grant these permissions.
+After the release of the patch, these permissions MUST be revoked.
+
+Because of the urgent nature of security patches, an _informal_ vote during the
+CloudEvents SDK meeting is enough to assign the security patch to the _sdk
+maintainer(s)_ who volunteers to take care of it.
+
+NOTE: this process is valid only for security patches, not for bugs or
+enhancements.
+
+### Handover to a new maintainer/group of maintainers
+
+If a project is not meeting the criteria 1, 2, 4, 5, 6, 7, 8, the community
+MAY decides to handover the project to a new maintainer/group of
+maintainers. The community can perform the handover to a new maintainer
+if all the following conditions are met:
+
+* A vote to add the new maintainer MUST have **already taken place** with
```

Comment:
Who initiates a vote if all maintainers have vanished?

Reply:
Perhaps the larger SDK maintainers should do it?

Reply:
oh, I guess it's implied that it's one of the total sdk maintainers.

Reply:
It's explained below

> A voting manager, a member of the community entitled to manage the vote. The voting manager MUST be part of people entitled to vote.